### DISCUSSION TITLE
Welcome to marsview Discussions!

### DISCUSSION THREAD

**Discussion by JackySoft** [JackySoft/marsview#37] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by elarbor** [comment]
其实你可以建一个组织，就放【无代码搭建平台】
![image](https://github.com/user-attachments/assets/fe162e0d-515a-4e86-8480-8f128e46a56f)

![image](https://github.com/user-attachments/assets/47b71b5d-47e4-4309-932c-83d0fab7acac)


这样你的【无代码搭建平台】仓库就可以放前端项目和后端项目（好几个项目）
![image](https://github.com/user-attachments/assets/63286fa8-c59b-42fe-827b-b89ca7230c0b)

![image](https://github.com/user-attachments/assets/07bfa473-0eb7-4b49-8ea1-b474b5d8d4a7)

**Comment by JackySoft** [comment]
好主意
